packages/amplify-frontend-ios/package.json

```diff
@@ -29,6 +29,10 @@
   },
   "jest": {
     "collectCoverage": true,
+    "collectCoverageFrom": [
```

Comment:
Sorry didn't think to check earlier, but most of the FE packages have source code in `lib` and the package root